mkdocs.yml

```diff
@@ -21,5 +21,6 @@ nav:
   - infrastructure.md
   - "Development":
       - "Docker": development/docker.md
+      - "Continuous Delivery": development/cd.md
```

Comment:
@kewne, see [this comment](https://github.com/open-craft/opencraft/pull/815/files#r663796037)